packages/create-directus-extension/lib/index.js

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 'use strict';
 
-import { inquirer } from 'inquirer';
+import inquirer from 'inquirer';
```

Comment:
This resolves the primary issue faced by #18183. This refers how it's imported in the API since both are also in ESM (although our current inquirer version is not ESM):

https://github.com/directus/directus/blob/c301cfbc82d1ce3ba8dad56800552c4899895b6e/api/src/cli/commands/schema/apply.ts#L4